modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_clothing.dm

```diff
@@ -50,6 +50,12 @@
 	icon_state = "policeofficerpatrolcap"
 	supports_variations_flags = CLOTHING_SNOUTED_VARIATION_NO_NEW_ICON
 	armor_type = /datum/armor/head_helmet
+	unique_reskin = list(
+		"Blue" = "policeofficercap",
+		"Sillitoe" = "policetrafficcap",
+		"German" = "policeofficerpatrolcap",
```

Comment:
Do we _have_ to label it as German?

Reply:
It was that or "Wanna-Be SS", if you have a better name, feel free to suggest it.